### ISSUE TITLE
"cd .." in the shell

### ISSUE THREAD

**Issue by BSzili** [salass00/smb2fs#19]
You can CD into the ".." directory in the Shell which does work, but ".." is added to the path every time you do this. Although "cd /" should be used instead, it would be nice to normalize the path in this case.
https://postimg.cc/zLPtpyqM

**Comment by salass00** [comment]
The problem is that "cd .." shouldn't be working in the first place. If FbxLockName2Path() in filesysbox is modified to return FALSE if the path contains any '.' or '..' components then it should fix this issue more or less entirely.

**Comment by BSzili** [comment]
You are right, it's a better solution to filter these out, and allow only Amiga-style paths.
